bin/index.js

```diff
@@ -170,7 +170,40 @@ updateLines();
 /*
  * Start speed test
  */
-var test = SpeedTestNet({maxTime: 10000});
+ 
```

Comment:
Indentation in this file should also be 2 spaces per level.

Reply:
done